index.js

```diff
@@ -89,6 +111,10 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+const myArrowFunction = () => {
+    return "I learned arrow functions today";
```

Comment:
all the code looks great!
Only additional thing you could do here is if you wanted to be slightly more concise, write this out as..
`const myArrowFunction = () => "This is a function expression!";`
^ You're allowed to write it like that when there's nothing else you need to do in the code block beside return. It's sort of a shorthand.